src/components/LayoutWrapperSideNav/LayoutWrapperSideNav.css

```diff
@@ -14,17 +14,13 @@
   box-shadow: var(--boxShadow);
 
   @media (--viewportLarge) {
-    padding: 112px 0 82px 15vw;
+    padding: 112px 0 82px calc((100% - 1056px) / 2 + 74px);
```

Comment:
This is part looks good 👍 

However, I noticed that subelements have still strange values. 

I'd propose that we also update `.tabs` in this file and `.link` in TabNav.

```css
@media (min-width: 1024px)
.LayoutWrapperSideNav__tabs__EKdgG {
  min-height: auto;
  flex-direction: column;
  margin-top: 24px; /* 28px is not divisible by baseline 8px */
  padding-top: 0;
}
```

Currently, in TabNav, we have correct height 56px (56/8=7). But it is constructed poorly:
```css
@media (min-width: 1024px)
.TabNav__link__1PgdT {
  border-bottom-width: 0px;
  margin-bottom: 14px;
  width: 200px;
  padding-top: 0;
  margin-top: 0;
  /* and padding-bottom: 10px; is coming from previous breakpoint */
}

```
It should be something like:
```css
@media (min-width: 1024px)
.TabNav__link__1PgdT {
  width: 200px;
  border-bottom-width: 0px;
  margin-top: 0;
  margin-bottom: 16px; /* 16/8=2 */

  /* Line-height is 32px and adding padding of 6px + 2px gives a total height of 40px which divisible by 8 */
  padding-top: 6px; /* Drop the text to adjust it to correct baseline */
  padding-bottom: 2px; 
}
```

Wurthermore, we also have problems with LayoutSideNavigation:
```css
@media (min-width: 1024px)
.LayoutSideNavigation__layoutWrapperMain__3jQ70 {
  padding: 116px 15vw 82px 82px; /* 116 / 8 = 14.5 => I think this should also be 112px;*/
}
```

That would mean that we should change `.heading` on TermsOfServicePage:
```css
@media (min-width: 768px)
.TermsOfServicePage__heading__2ni4I {
  margin: 8px 0 24px 0; /* currently, this is "margin: 2px 0 23px 0;" */
}
```
and in several other pages need similar adjustment since they use `LayoutSideNavigation`: 
- PrivacyPolicy
- ContactDetailsPage
- InboxPage
- PasswordChangePage
- PayoutPreferencesPage
- ProfilePage